packages/containers/src/ActionBar/ActionBar.connect.js

```diff
@@ -0,0 +1,36 @@
+import { cmfConnect } from '@talend/react-cmf';
+import { ActionBar } from '@talend/react-components';
+import getRenderers from '../renderers';
+
+export function mapStateToProps(state, { actionIds }) {
+	const props = {
+		renderers: getRenderers(),
```

Comment:
im starting to think that something along the name `getActionRender` could be nice to use

Reply:
it's not only for Action in fact, it will be also used on List and SidePanel